ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1694,13 +1694,17 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     }
 
     func separatorLeadingInset(for type: SeparatorType) -> CGFloat {
-        guard type == .inset else {
+        switch type {
+        case .none:
             return 0
+        case .inset:
+            let baseOffset = TableViewCell.selectionModeAreaWidth(isInSelectionMode: isInSelectionMode,
+                                                              selectionImageMarginTrailing: tokenSet[.selectionImageMarginTrailing].float,
+                                                              selectionImageSize: tokenSet[.selectionImageSize].float)
+            return baseOffset + paddingLeading + tokenSet[.customViewDimensions].float + tokenSet[.customViewTrailingMargin].float
+        case .full:
+            return -safeAreaInsets.left
```

Comment:
in RTL do you need to check safeareainsets.right? easier to test if the iphone x bezel is in the other side?

Reply:
Sanity check:

Left bezel:
<img width="846" alt="image" src="https://user-images.githubusercontent.com/22566866/185009034-ed67e92a-3324-406c-a2eb-9f0628eda1fc.png">

Right bezel:
<img width="842" alt="image" src="https://user-images.githubusercontent.com/22566866/185009080-e81f16e4-0719-40dd-b275-9150b8ae957e.png">

Reply:
i don't see a code update?

Reply:
Ok new sanity check with new changes:

![image](https://user-images.githubusercontent.com/22566866/185215870-ada8db30-100b-4891-9334-1f8c0533cbac.png)
![image](https://user-images.githubusercontent.com/22566866/185215939-29f3edd3-396c-40a6-9a3d-2297cdedfdde.png)